test/server/progress-option.test.js

```diff
@@ -29,7 +29,7 @@ describe('progress', () => {
         let foundProgress = false;
         let foundProfile = false;
         calls.forEach((call) => {
-          if (call[0].includes('0% compiling')) {
+          if (call[0].includes('2%')) {
```

Comment:
The previous webpack@next of beta was able to output 0% but the new version can't output this. I'll investigate this but anyway, this PR is okay.(progress feature can work).